public/examples/google-pay.html

```diff
@@ -36,10 +36,11 @@
         return googlePay;
       }
 
-      async function createPayment(token) {
+      async function createPayment(token, verificationToken) {
```

Comment:
add a comment like on top of this like the one we have in other verification token using examples
```
// verificationToken can be undefined, as it does not apply to all Google Pay payments.
```

Reply:
Can it be undefined? I think that we create a verification token on all calls to `verifyBuyer`. What we do not do is add a 3DS challenge to all verifications.

Reply:
It will be undefined only if `verifyBuyer` is not called. In other samples we control the `verifyBuyer` calls with a flag and that is why a similar comment was added but you are right in this example we do not make that additional control and this change makes `verifyBuyer` calls for each payment. My comment can be discarded